docs/sql-ref-syntax-qry-explain.md

```diff
@@ -19,4 +19,157 @@ license: |
   limitations under the License.
 ---
 
-**This page is under construction**
+### Description
+
+The `EXPLAIN` statement provides the execution plan for the statement. 
+By default, `EXPLAIN` provides information about the physical plan.
+`EXPLAIN` does not support 'DESCRIBE TABLE' statement.
+
+
+### Syntax
+{% highlight sql %}
+EXPLAIN [EXTENDED | CODEGEN] statement
+{% endhighlight %}
+
+### Parameters
+
+<dl>
+  <dt><code><em>EXTENDED</em></code></dt>
+  <dd>Generates Parsed Logical Plan, Analyzed Logical Plan, Optimized Logical Plan and Physical Plan.</dd>
+</dl> 
+
+<dl>
+  <dt><code><em>CODEGEN</em></code></dt>
+  <dd>Generates code for the statement, if any.</dd>
```

Comment:
You can also refer the documentations at https://github.com/apache/spark/blob/master/sql/core/src/main/scala/org/apache/spark/sql/execution/ExplainMode.scala#L29-L57